.github/workflows/node-ci.yml

```diff
@@ -195,7 +195,11 @@ jobs:
 
     upload_coverage:
         name: Upload Coverage
-        runs-on: ubuntu-latest
+        runs-on: ${{ matrix.os }}
+        strategy:
+            matrix:
+                os: [ubuntu-latest]
+                node-version: [20.x]
```

Comment:
Note that on line 210 below, we reference `matrix.node-version` but until this PR that didn't exist. According to Github's docs, it will use an empty string if a matrix value is missing. Which would explain the missing value in the cache keys.... I hope.